src/window/grid.rs

```diff
@@ -214,6 +214,8 @@ unsafe extern "system" fn callback(
                             let _ = sender.send(Message::CloseWindows);
                         }
                     }
+
+                    grid.unselect_all_tiles();
```

Comment:
Good call 👍 this is visually more appealing as well, as the blue is gone after a selection is made